kong/plugins/method-restriction/handler.lua

```diff
@@ -0,0 +1,30 @@
+local BasePlugin = require "kong.plugins.base_plugin"
+local responses = require "kong.tools.responses"
+
+local MethodRestrictionHandler = BasePlugin:extend()
+
+MethodRestrictionHandler.PRIORITY = 940
+
+local function checkMethod(conf, method)
+  for i = 1,#conf.methods do
```

Comment:
I think for now, you best use this utility function (which does the same iteration); https://github.com/Mashape/kong/blob/master/kong/tools/utils.lua#L180

Fixing the `enum` type in the DAO is on the todo list, and then we'll provide all enum types automatically with a reverse entry probably. Using that function will enable to to quickly search-and-replace all occurrences, including this one, once fixed.